cf/data/utils.py

```diff
@@ -563,71 +563,77 @@ def scalar_masked_array(dtype=float):
 def conform_units(value, units, message=None):
     """Conform units.
 
-    If *value* has units defined by its `Units` attribute then
+        If *value* has units defined by its `Units` attribute then
```

Comment:
All of the line indentation has been messed up with the changes here on the docstring. For consistency with other docstrings we should revert this extra level of indentation across the whole docstring, including the 'Examples' section...

Reply:
I think that was emacs being a bit too clever/daft, perhaps? Fixed in 93ffa56cc0b617b778fa0cf32a4c9c2bbad8addd